Server/utils/validators/contentValidator.js

```diff
@@ -0,0 +1,25 @@
+const { check, body } = require('express-validator');
+
+exports.contentValidator = [
+    check('title')
+        .notEmpty().withMessage('Title is required'),
+    check('description')
+        .notEmpty().withMessage('Description is required'),
+    check('type')
+        .notEmpty().withMessage('Content Type is required')
+        .isIn(['link', 'file', 'text']).withMessage('Type must be one of "link", "file", or "text"'),
+    check('lectureId')
+        .notEmpty().withMessage('Lecture ID is required')
+        .isInt().withMessage('Lecture ID must be an integer'),
```

Comment:
you shouldn't validate your lectureId anymore because you are getting it from the url params